recipes/ribocode/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: ribocode
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz # [py2k]
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz # [py2k]
+  md5: e05ef843da351c4588e02bad3d5c3d65 # [py2k]
+
+requirements:
+  build:
+    - python # [py27]
```

Comment:
can you please remove the pinning here (# [py27]) ... you skip py3k build already.